src/app/components/sidenav/categories-select/categories-select.component.html

```diff
@@ -12,6 +12,8 @@
                 mat-button
                 hideToggle
                 class="sub-catigory"
+                [routerLink]="['products-list']"
```

Comment:
Хм, если тут только строка, почему бы не сделать так:
```suggestion
                routerLink="/products-list"
```

И почему ссылку решил сделать относительной, а не абсолютной?

Reply:
Лучше сделать абсолютной